### PR TITLE
feature: acceptance tests CLI

### PR DESCRIPTION
**Pull request by geclos**
Runs an acceptance test in CI that starts a default project and runs the local server with the production build of the CLI.

### CODE REVIEW

File: packages/cli/tests/e2e/start.cjs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  // Logic to decide when and what to send based on the output
+  if (!projectCreated && output.includes("What's the name of your project?")) {
+    projectCreated = true
+    startProcess.write('test-project\n')
+  }
+
+  if (!templateChosen && output.includes('Pick a template')) {
+    templateChosen = true
+    startProcess.write('\n')
+  }
```
Comment (andresgutgon) [inline packages/cli/tests/e2e/start.cjs:29]: Beautiful

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          key: ${{ runner.os }}-turbo-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-turbo-
+
+      - name: Install dependencies
+        run: |
+          pnpm install
+
+      - name: Use Turbo to build affected packages
+        run: pnpm turbo run build --cache-dir=.turbo --filter='./packages/cli'
```
Comment (andresgutgon) [inline .github/workflows/e2e.yml:64]: Only cli needs to be built?

Comment (geclos) [reply .github/workflows/e2e.yml:64]: yup

Comment (geclos) [reply .github/workflows/e2e.yml:64]: yes

**Review by andresgutgon — DISMISSED** [review]

File: .gitignore [hunk 3]
```diff
@@ -33,3 +33,4 @@ yarn-error.log*
 
 sites/**/*
 !sites/package.json
+test-project
```
Comment (andresgutgon) [inline .gitignore:36]: Why not create it inside `sites`? Already ignored folder

**Review by andresgutgon — APPROVED** [review]

**Pull request merged**